csrc/include/sampling.cuh

```diff
@@ -1,6 +1,23 @@
 // SPDX-License-Identifier: MIT
 // Copyright (c) 2024, Advanced Micro Devices, Inc. All rights reserved.
 
+/*
+ * Copyright (c) 2025 by SGLang team.
```

Comment:
This file is developed by FlashInfer team, you should probably remove this line.